content/docs/faq-internals.md

```diff
@@ -1,23 +1,23 @@
 ---
 id: faq-internals
-title: Virtual DOM and Internals
+title: वास्तविक DOM और इंटरनल्स 
 permalink: docs/faq-internals.html
 layout: docs
 category: FAQ
 ---
 
-### What is the Virtual DOM? {#what-is-the-virtual-dom}
+### वास्तविक DOM क्या है? {#what-is-the-virtual-dom}
 
-The virtual DOM (VDOM) is a programming concept where an ideal, or "virtual", representation of a UI is kept in memory and synced with the "real" DOM by a library such as ReactDOM. This process is called [reconciliation](/docs/reconciliation.html).
+वास्तविक DOM (VDOM) एक प्रोग्रामिंग कांसेप्ट है जहां एक आदर्श, या "वास्तविक", UI का रिप्रजेंटेशन मेमोरी में रखा जाता है और ReactDOM जैसी लाइब्रेरी द्वारा "वास्तविक" DOM के साथ सिंक में रखा जाता है। इस प्रक्रिया को [reconciliation](/docs/reconciliation.html) कहते हैं।
 
-This approach enables the declarative API of React: You tell React what state you want the UI to be in, and it makes sure the DOM matches that state. This abstracts out the attribute manipulation, event handling, and manual DOM updating that you would otherwise have to use to build your app.
+यह तरीका React कि वर्णनात्मक API को सक्षम बनाता है: आप React को यह बताएं कि आप किस state में UI को चाहते हैं और यह सुनिश्चित करता है कि DOM उस अवस्था से मेल खाता है। यह एट्रिब्यूट मैनीपुलेशन, ईवेंट हैंडलिंग और मैन्युअल DOM अपडेट को अमूर्त करता है वरना जिसका आपको अपने ऐप को बनाने के लिए उपयोग करना पढता।
```

Comment:
`आप React को यह बताएं कि आप किस state में UI को चाहते हैं और यह सुनिश्चित करता है कि DOM उस अवस्था से मेल खाता है।`

can you revisit this translation. sounds odd.

Reply:
Okay